### DISCUSSION TITLE
Raising PR's & Permissions

### DISCUSSION THREAD

**Discussion by avalon60** [TomSchimansky/CustomTkinter#2120] [General]
Is it no longer possible to raise PR's with the new 2FA?

If it is how do I go about this - I am getting permission errors:

![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/7255e553-21df-4eee-94cc-4cb9bfc390bc)

Thanks in advance.

**Comment by Akascape** [comment]
Use github web version...
